### PR TITLE
add image viewbox clipping

### PR DESCRIPTION
**Pull request by luke1188**
adds back viewbox clipping

### CODE REVIEW

File: src/runtime/mod.rs [hunk 1]
```diff
@@ -44,6 +44,11 @@ impl Composition {
         let composition = import::conv_animation(source);
         Ok(composition)
     }
+    /// Returns a t value for the specified time in seconds.
+    pub fn frame_for_time(&self, secs: f64) -> f64 {
+        let frame = secs * self.frame_rate;
+        frame % self.frames.end
+    }
```
Comment (simbleau) [inline src/runtime/mod.rs:51]:
```suggestion
```

This didn't have anything to do with viewbox clipping

**Review by simbleau — CHANGES_REQUESTED** [review]
It was probably a mistake to include "frame_for_time" so let's remove that, then we can merge this.

**Review by simbleau — APPROVED** [review]
Looks good.

**Comment by DJMcNab** [comment]
I've re-run the CI, as the failure looked spurious

@simbleau, how do you want this PR to be merged? I'd suggest that you just press the green button yourself

**Comment by simbleau** [comment]
I'll merge it tomorrow, when i have time to add to the changelog and release a patch version (0.1.1)

I consider the lack of viewbox clipping to be a bug, since many Lotties depend on it.

**Pull request merged**